components/Header.js

```diff
@@ -0,0 +1,92 @@
+import React, { useMemo } from "react";
+import { Image } from "expo-image";
+import { StyleSheet, Pressable, Text, View } from "react-native";
+import { FontSize, FontFamily, Color, Padding } from "../GlobalStyles";
+
+const getStyleValue = (key, value) => {
+  if (value === undefined) return;
+  return { [key]: value === "unset" ? undefined : value };
+};
+const Header = ({
+  invoiceTitle,
+  backButtonPosition,
+  backButtonTop,
+  backButtonLeft,
+  iNVOICESAlignItems,
+  iNVOICESWidth,
```

Comment:
nit:
lets keep the coding conventions yaa, using camel case for variables
```suggestion
  invoicesAlignItems,
  invoicesWidth,
```